### PR TITLE
disable case tiles when in landscape mode of form entry

### PR DESCRIPTION
**Pull request by amstone326**

### CODE REVIEW

**Comment by phillipm** [comment]
Are we sure we want to take this approach to increasing the visible form entry area while in landscape mode? What about making persistent case tiles collapse-able and then auto-collapsing while in landscape mode?

File: app/src/org/commcare/android/framework/BreadcrumbBarFragment.java [hunk 1]
```diff
@@ -360,6 +363,10 @@ public void onResume() {
             }
         }
     }
+
+    private boolean inLandscapeFormEntry(Activity a) {
+        return (a instanceof FormEntryActivity) && getResources().getConfiguration().orientation == Configuration.ORIENTATION_LANDSCAPE;
```
Comment (phillipm) [inline app/src/org/commcare/android/framework/BreadcrumbBarFragment.java]: This function can be made static if you call `getResources()...` on the `a` argument

**Comment by amstone326** [comment]
@phillipm Not sure what you mean by "making persistent case tiles collapse-able and then auto-collapsing while in landscape mode" or how it's different from this. Can you explain?

**Comment by ctsims** [comment]
Case tiles are still very custom, so I'm inclined to not make any drastic
UI changes based on them without thinking hard about it. Anyone using case
tiles already knows how it's affecting their screens/projects, so I kind of
don't want to be making big changes to their experience.

No strong feelings though. Most case tile users are locked in portrait mode.

**Comment by amstone326** [comment]
Ok fair, I think I'm gonna just close this for now then.

**Pull request closed without merging**